.github/ISSUE_TEMPLATE/staging-review.md

```diff
@@ -66,6 +66,8 @@ https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platfo
 
 ### Content artifacts 
 
+- [ ] Your product’s content source of truth
+- [name of content of truth, for example, "Invision Prototypes"](link here)
```

Comment:
you can probably remove the 2nd line... I think Peggy originally had it as a comment. The first line alone is sufficient.